cosmwasm/enclaves/shared/block-verifier/src/submit_block_signatures.rs

```diff
@@ -0,0 +1,207 @@
+use std::slice;
+
+use tendermint_proto::Protobuf;
+
+use sgx_types::sgx_status_t;
+
+use enclave_utils::{validate_const_ptr, validate_input_length, validate_mut_ptr};
+use log::error;
+
+use log::debug;
+
+use tendermint::validator::Set;
+
+macro_rules! unwrap_or_error {
```

Comment:
I think the name is very confusing I would rather call it unwrap_or_return instead.